spec/validation/loader_spec.rb

```diff
@@ -47,9 +47,66 @@
       end.to_h
     end
 
+    let :example_plugin_configure_questions_hash do
+      configure_sections.map do |section|
+        [
+          section, [{
```

Comment:
Can this be broken up to something like:
```
key = section
value = **blah**
[key, value]
```
In it's current form it isn't obvious how this will be converted into an array due to the size complexity of the array object.